_docs/concepts/what-is-istio/overview.md

```diff
@@ -97,13 +97,13 @@ This loose coupling allows Istio to run on multiple environments
 (e.g., Kubernetes, Consul/Nomad) while maintaining the same operator
 interface for traffic management.
 
-### Istio-Auth
+### Security
```

Comment:
I don't think we want to use "Security" here. The list is about individual components "Mixer", "Pilot", "Envoy". Security is not a proper noun.

This kind of situation is why I was mentioning that we really should name Istio Auth or Istio CA something specific.

Reply:
Also note that the diagram on this page says Istio-Auth.